### DISCUSSION TITLE
Examples of rtsp proxy

### DISCUSSION THREAD

**Discussion by adzmailru** [bluenviron/mediamtx#3485] [Questions] [answered]
### Question

Dear, Gurus!
Please help me!
I need examples of configuring the rtsp proxy!!! Scheme: rtsp local cam 192.168.10.10 -> rtsp proxy mediamtx (server 45.70.72.191) -> internet.
Please share links to examples of the mediamtx.yml configuration file!!!

**Comment by moenodedev** [comment]
The readme has examples for proxying rtsp
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#proxy-requests-to-other-servers

**Comment by adzmailru** [comment, marked as answer]
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#rtsp-cameras-and-servers

sсheme: rtsp cam -> wireguard -> wg server +rtsp proxy mediamtx -> port of server

paths:
  cam100:
     source: rtsp://user:pass@192.168.10.10:554
     sourceOnDemand: yes

inet show -> rtsp://45.70.72.191:8554/cam100

[a comment by an automated account is not shown]
